ff/serving/provider/online.go

```diff
@@ -221,3 +357,75 @@ func (table redisOnlineTable) Get(entity string) (interface{}, error) {
 	}
 	return result, nil
 }
+
+// CASSANDRA SET ENTITY
+func (table cassandraOnlineTable) Set(entity string, value interface{}) error {
+
+	table.session.SetConsistency(gocql.One)
+
+	//Get table key
+	key := table.key
+	var keyName string
+	keyName = sn.Custom(key.String(), "[^a-zA-Z0-9_]")
+
+	//Get table Name
+	var tableName string
+	tableName = fmt.Sprintf("%s.table%s", key.Keyspace, sn.Custom(key.Feature, "[^a-zA-Z0-9_]"))
+
+	//Set entity
+	val := fmt.Sprintf("%v", value)
+	query := fmt.Sprintf("INSERT INTO %s (%s, %s) VALUES (?, ?)", tableName, keyName, string(table.valueType))
+	err := table.session.Query(query, entity, val).WithContext(ctx).Exec()
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// CASSANDRA GET ENTITY
+func (table cassandraOnlineTable) Get(entity string) (interface{}, error) {
+
+	//Get table key
+	key := table.key
+	var keyName string
+	keyName = sn.Custom(key.String(), "[^a-zA-Z0-9_]")
+
+	//Get table Name
+	var tableName string
+	tableName = fmt.Sprintf("%s.table%s", key.Keyspace, sn.Custom(key.Feature, "[^a-zA-Z0-9_]"))
+
+	//Get table value type
+	var valName string
+	valName = string(table.valueType)
+
+	//Get value
+	var val string
+	query := fmt.Sprintf("SELECT %s FROM %s WHERE %s = '%s'", valName, tableName, keyName, entity)
+	err := table.session.Query(query).WithContext(ctx).Scan(&val)
+	if err != nil {
+		return nil, &EntityNotFound{entity}
+	}
+
+	switch table.valueType {
+	case Int:
+		return strconv.Atoi(val)
+	case Int64:
+		return strconv.ParseInt(val, 10, 64)
+	// Set float 32 as strconv only outputs float 64
+	case Float32:
+		var result64 float64
+		var result32 float32
+		var err error
+		result64, err = strconv.ParseFloat(val, 32)
+		result32 = float32(result64)
```

Comment:
if ParseFloat returns a value for err, would result64 exist and be convertible here?